### ISSUE TITLE
proposal to relax log-crw event code matching

### ISSUE THREAD

**Issue by turbach** [kutaslab/mkpy#22]
Add an option to the crw, log reader to allow crw, log code mismatches.

```
Parameter
----------
log_evcodes : str ("aligned", "strict", "any", "none")
    "aligned" (default=current behavior) requires 1-1 match on data samples
    (crw_tick), but log and crw code values may differ. 

    "strict" requires 1-1 match by crw_tick and value
   
     "any" loads log event codes, warns if log code crw_ticks overrun the crw data.
    
     "none" ignores the log file, if any, all log event codes set to 0
```

Use cases:

`aligned` is the normal operation where crw and log codes align but the log may have been `logpoke`-ed.

`strict` if circumstances allow or require perfect crw-to-log event alignment.

`any` loads crw  and whatever log codes it can for the experimenter to deal with offline.

`none` permits conversion of crw-only data and allows recovery of data for corrupted or lost logs.

**Comment by astoermann** [comment]
I have run into issues where the crw is jumbled from a dig pause. I can provide a datapath to the example if you like, but I don't think it falls into any of these categories as being a fix, or if it even needs to.

**Comment by turbach** [comment]
closed by [v0.2.3](https://github.com/kutaslab/mkpy/tree/v0.2.3)
